gateway/server.go

```diff
@@ -33,6 +35,27 @@ func main() {
 
 	log.Printf("Binding to external function provider: %s", config.FunctionsProviderURL)
 
+	var credentials *handlers.BasicAuthCredentials
+
+	if config.UseBasicAuth {
+		userPath := "/var/secrets/basic_auth_user"
```

Comment:
You need to check the `/run/secrets/*` files when running on Swarm or the gateway will fail to initialise.

Reply:
In which case should I make the secret path a config item too?

Reply:
Could just check both, its only going to occur once on startup so shouldn't be an issue? Saves having to expose more config to the user.

Reply:
@stefanprodan is there such a thing as a standard location for secrets on Kubernetes?

Reply:
There is no  standard location, usually ok Kubernetes you would mount secrets as ENV vars not files on disk.